### PR TITLE
Final tweaks for Kusama PrePos.

### PR DESCRIPTION
**Pull request by gavofyork**
PrePoS parameters:
- Enabled modules Technical Committee & Membership, as well as Council Elections
- Staking: Unbonding duration: 7 days
- Democracy: Launch period, Voting period: 7 days
- Democracy: Enactment period: 8 days
- Democracy: Emergency voting period: 3 hours
- Treasury: Spend period: 6 days
- Treasury: Unspent burn amount: 1%
- Treasury: Approval required: 60%

### CODE REVIEW

**Comment by gavofyork** [comment]
i was going to open up democracy and council once it was clear that elections works.

**Comment by gavofyork** [comment]
One more thing that might be helpful...

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -130,7 +130,8 @@ impl SignedExtension for OnlyStakingAndClaims {
 	{
 		match call {
 			Call::Staking(_) | Call::Claims(_) | Call::Sudo(_) | Call::Session(_)
-				| Call::ElectionsPhragmen(_)
+				| Call::ElectionsPhragmen(_) | Call::TechnicalMembership(_)
+				| Call::TechnicalCommittee(_)
```
Comment (andresilva) [inline runtime/src/lib.rs]: Do we need to enable `Call::Council(_)` as well?

Comment (andresilva) [reply runtime/src/lib.rs]: Nevermind, I see that getting into the council will go through `ElectionsPhragmen`

Comment (gavofyork) [reply runtime/src/lib.rs]: not yet - we can do that once we actually want to have the council do stuff. but i'd prefer to test things one at a time first.

**Pull request merged**